stylesheets/components/grid.css

```diff
@@ -0,0 +1,24 @@
+.grid-col-2{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    border: 1px solid black;
```

Comment:
just a suggestion...you can remove the border style from grid class and create a seperate class only for border (like .border-1 ). This will make your grid class more flexible for customization while using it in projects.

Reply:
Implemented in next commit